### PR TITLE
Move production queue factors from UserString to OptionsDB

### PR DESCRIPTION
**Pull request by dbenage-cx**
Moves the settings to adjust the production queue allocations into OptionsDB.

Partial proposal towards #907

### CODE REVIEW

**Comment by Vezzra** [comment]
@dbenage-cx: you know, one perks of having write/push access now is that you can set the labels & milestones as you see fit. 😉

**Comment by dbenage-cx** [comment]
TY, will get these lined up :D

**Comment by Vezzra** [comment]
Feel free to change the assignments I did in the meantime wherever you think something else is more appropriate.

File: Empire/Empire.cpp [hunk 1]
```diff
@@ -33,51 +33,55 @@ namespace {
...
-// Both factors have a minimum value of 0.0 and a maximum value of 0.3.
-
-// Making the frontloaded factor greater than zero increases the per-turn allocation cap by the specified percentage (so it always spreads the extra allocation across all turns).
-// Making the topping-up option nonzero allows the final turn allocation cap to be increased by the specified percentage of the total cost, if needed (and then subject to 
-// availability of course). They can both be nonzero, although to avoid that introducing too much interaction complexity into the minimum build time safeguard for topping-up, 
-// the topping-up percentage will be reduced by the frontloading setting. 
-
-// Note that for very small values of the options (less than 5%), when dealing with very low cost items the effect/protection may be noticeably less than expected because of
-// interactions with the ProductionQueue Epsilon value (0.01)
+    void AddOptions(OptionsDB& db) {
```
Comment (geoffthemedio) [inline Empire/Empire.cpp:36]: These are gameplay mechanics modifications, and thus not appropriate to put in the OptionsDB.

Comment (dbenage-cx) [reply Empire/Empire.cpp:36]: Ok, shelving this for pending discussion.

**Pull request closed without merging**